packages/common-tests/browser-tests/roles/allCookiesUser.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  ConsentModal,
+  getEnvUrl,
+} from 'events-helsinki-common-tests/browser-tests';
+import { Role } from 'testcafe';
+
+export const acceptAllCookies = async () => {
+  console.log('allCookiesUser: acceptAllCookies!');
+  const cookieConsentModal = new ConsentModal();
+  await cookieConsentModal.isOpened();
+  await cookieConsentModal.clickAcceptAllCookies();
+};
+
+const userAcceptingAllCookies = Role(
+  getEnvUrl('/'),
+  async () => {
+    await acceptAllCookies();
+  },
+  { preserveUrl: false }
+);
```

Comment:
There are still some issues with this. Sometimes, especially during the first run, the Testcafe miss-clicks the allow all cookies -button and instead of closing the cookie consent modal, it just minimizes it and then it blocks some clicks that we are expecting to happen during the tests.